src/applications/ezr/utils/helpers/emergencyContactUtils.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Helper to get the item name for the emergency contact.
+ * @param {Object} item - The emergency contact item containing fullName.
+ * @returns {String} - Returns the full name (first and last name) or an empty string if data is missing.
+ */
+export const getItemName = item => {
+  if (item?.fullName?.first && item?.fullName?.last) {
+    return `${item.fullName.first} ${item.fullName.last}`;
+  }
+  return ''; // Fallback if data is missing
+};
+
+/**
+ * Helper to get the card description, typically the primary phone number.
+ * @param {Object} item - The emergency contact item containing primaryPhone.
+ * @returns {String} - Returns the primary phone number or an empty string if not available.
+ */
+export const getCardDescription = item => `${item?.primaryPhone || ''}`;
+
+/**
+ * Helper to generate the delete title text for the modal.
+ * @returns {String} - Returns the delete confirmation title for the emergency contact.
+ */
+export const getDeleteTitle = () => 'Delete this emergency contact?';
+
+/**
+ * Helper to generate the confirmation text for deleting the emergency contact.
+ * @returns {String} - Returns the delete confirmation text for the emergency contact.
+ */
+export const getDeleteYes = () => 'Yes, delete this emergency contact';
+
+/**
+ * Helper to generate the delete description text.
+ * @param {Object} item - The emergency contact item containing fullName.
+ * @returns {String} - Returns the delete description, including the first and last name or a fallback if the names are missing.
+ */
+export const getDeleteDescription = item => {
```

Comment:
You can use the `replaceStrValues` helper for this. If you add a `%s` to the string and bring it in from the content.json file, you can clean the splicing of the first and last name up. [You can reference the use of this here.](https://github.com/department-of-veterans-affairs/vets-website/blob/5921e8176c73f488b0e5d95e30860faee83d416e/src/applications/ezr/components/FormFields/InsurancePolicyList.jsx#L92)

Reply:
Great call out this refactor tidbit actually led to getting a piece of functionality surrounding the delete description always defaulting to the default value and not entering the if block. After reworking this and utilizing the replaceStrValues helper I was able to get the if block working correctly finally. Thanks Matt!